packages/packages.go

```diff
@@ -81,9 +81,15 @@ type Packages struct {
 
 // PkgInfo describes a package.
 type PkgInfo struct {
-	Name, Arch, Version string
+	Name, Version string
+	Arch          OSArch
+	Source        Source
+}
 
-	Source Source
+// OSArch describes a OS Architecture specified for package.
+type OSArch struct {
+	StandardArch string
```

Comment:
I'm thinking of how to make a bit more readable usage of this two fields, i think that might be reasonable to create two methods 

```
func (a Architecture) Raw() string {
    return a.RawArch
}

func (a Architecture) Architecture() string {
    return a.StandardArch
}
```

This will give a signal that raw architecture is rather additional not main data stored in the structure and that in most scenarios standard architecture should be used

Reply:
As well it might be a good idea to comment fields as current names is not straightforward for me